astropy/table/tests/test_column.py

```diff
@@ -229,7 +229,7 @@ def test_item_access_type(self, Column):
         Tests for #3095, which forces integer item access to always return a plain
         ndarray or MaskedArray, even in the case of a multi-dim column.
         """
-        integer_types = (int, np.int)
+        integer_types = [int]
```

Comment:
This one was a bit weird. Was it indentded to test `np.int_`?

Reply:
I looked back at #3095, and, yes, this is definitely meant to test all possible integer types, so replace this with `np.int_`. (The actual code looks for subclasses of `int` and `np.integer`)